libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Input/ConfirmInput.cs

```diff
@@ -178,7 +178,9 @@ protected override async Task<IActivity> OnRenderPromptAsync(DialogContext dc, I
 
         private string DetermineCulture(DialogContext dc)
         {
-            var culture = PromptCultureModels.MapToNearestLanguage(dc.Context.Activity.Locale ?? DefaultLocale?.GetValue(dc.State));
+            // Note: Default locale will be considered for deprecation as part of 4.13.
+            var candidateLocale = dc.GetLocale() ?? DefaultLocale?.GetValue(dc.State);
+            var culture = PromptCultureModels.MapToNearestLanguage(candidateLocale);
```

Comment:
nit: this code is very similar to the "choice input" - maybe some internal helper - though perhaps its sufficiently different to make that tricky?